src/components/Status/Status.js

```diff
@@ -19,14 +20,16 @@ Status.propTypes = {
   children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
 };
 
-export const OverlayStatus = ({ icon, text, overlay }) => (
-  <Status icon={icon}>
-    <OverlayTrigger overlay={overlay} placement="right" trigger={['click']} rootClose>
-      <Button className="kubevirt-status__button" bsStyle="link">
-        {text}
-      </Button>
-    </OverlayTrigger>
-  </Status>
+export const OverlayStatus = ({ icon, header, children }) => (
+  <Popover position="right" headerContent={header} bodyContent={children}>
+    <span>
```

Comment:
`span` is unnecessary here, isn't it ?

Reply:
When I remove the `<span>` the trigger component is only the `icon`, and pressing the `button` element does not trigger the popover :-(

This may be a bug in the `popover` element, but "adding span" fix/hack is easy ... and solves the problem.

@rawagner WDYT, can we do it a nicer way, can we ignore this odd behaviur from popover, and assume it will be fixed upstream ?

Screenshot of problem:
![Peek 2019-05-23 14-27](https://user-images.githubusercontent.com/2181522/58249373-5f91b980-7d67-11e9-94d6-a75857142ab3.gif)

Reply:
ah, ok. Popover expects one child. Can you add style to the span element to show pointer cursor ?

Reply:
wow, didn't notice the pointer shape, thanks :cake: